visual_studio/libmatio/libmatio.vcxproj

```diff
@@ -165,18 +165,8 @@
       <RuntimeLibrary>MultiThreadedDLL</RuntimeLibrary>
       <FunctionLevelLinking>true</FunctionLevelLinking>
       <WarningLevel>Level3</WarningLevel>
-      <DebugInformationFormat>
-      </DebugInformationFormat>
+      <DebugInformationFormat>ProgramDatabase</DebugInformationFormat>
     </ClCompile>

```

Comment:
I believe this is the reason why there are hundreds of linker warnings. Better to not generate PDB then.